dashboard/lib/tasks/setup_or_migrate.rake

```diff
@@ -14,6 +14,7 @@ namespace :db do
 
   timed_task_with_logging :setup_or_migrate do
     db_exists = database_exists?
+    ENV['db_read_timeout'] = 300
```

Comment:
Might be useful to adopt our convention of using all caps for environment variable names (`DB_READ_TIMEOUT`).